packages/@uppy/companion/src/server/provider/dropbox/index.js

```diff
@@ -215,7 +215,7 @@ class DropBox extends Provider {
   _error (err, resp) {
     if (resp) {
       const fallbackMessage = `request to ${this.authProvider} returned ${resp.statusCode}`
-      const errMsg = resp.body.error_summary ? resp.body.error_summary : fallbackMessage
+      const errMsg = (resp.body || {}).error_summary ? resp.body.error_summary : fallbackMessage
```

Comment:
this change can be ignored. I accidentally checked out of [this branch](https://github.com/transloadit/uppy/pull/2264/files)